pkg/cluster/cert.go

```diff
@@ -39,8 +45,14 @@ func CreateSelfSignedCertificate(ctx context.Context, c client.Client, secretNam
 		return errApply
 	}
 
-	if errGen := generateCertSecret(ctx, c, certSecret); errGen != nil {
-		return fmt.Errorf("failed update self-signed certificate secret: %w", errGen)
+	fieldOwner := fmt.Sprintf("%s/%s", resources.PlatformFieldOwner, CertFieldOwnerSuffix)
```

Comment:
this can probably be just a constant ?

Reply:
Yep, good call. This is fixed in the latest commit.